v2/bindings/csharp/SnapPluginLibCs/SnapPluginLib/SnapPluginLib.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>netstandard1.4</TargetFramework>
+	    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
why ?

Reply:
This library is new. Why do u want to write it in legacy framework? do u have any business needs for that?

Reply:
need to support windows server 2012 out of the box.

Reply:
ok, i will revert it to netstandard1.4

Reply:
This c# lib code will be used and shipped by some c# snap plugin like apiPoller. I do not see reason to support legacy framework for that package